src/consensus/params.h

```diff
@@ -55,7 +28,6 @@ struct Params {
      */
     uint32_t nRuleChangeActivationThreshold;
     uint32_t nMinerConfirmationWindow;
```

Comment:
`nRuleChangeActivationThreshold` and `nMinerConfirmationWindow`  seem to be BIP9 parameters.